packages/SwingSet/docs/dynamic-vats.md

```diff
@@ -36,7 +36,7 @@ Then turn this into a bundle:
 
 ```js
 import bundleSource from '@agoric/bundle-source';
-
+ 
```

Comment:
yikes! don't add meaningless white space.

Reply:
heh, ok, will remove. My editor's markdown-mode shows triple-backquote blocks as literal code iff there are no blank lines in the middle. I was thinking that was a property of Markdown, but maybe it's just a bug in my editor.